### PR TITLE
Rename SourceContract to ProcessedSource

### PR DESCRIPTION
**Pull request by gnidan**
See preview [here](https://trufflesuite.github.io/artifact-updates/contrib/trufflesuite/processed-sources/data-model.html)

### CODE REVIEW

File: docs/uml/macros.iuml [hunk 1]
```diff
@@ -392,17 +392,15 @@ skinparam defaultMonospacedFontName Andale Mono
     }
 !endif
 
-!ifdef SHOW_SOURCE_CONTRACT
+!ifdef SHOW_PROCESSED_SOURCE
```
Comment (fainashalts) [inline docs/uml/macros.iuml:395]: Will `processedSource` not have a name field? I believe it still has one in the schema so I wanted to check in on this!

**Review by fainashalts — APPROVED** [review]
One comment/question but otherwise LGTM!

File: docs/uml/macros.iuml (hunk 1 above)
Comment (gnidan) [reply docs/uml/macros.iuml:395]: Hm I don't think it needs one (not sure what it'd be useful for on its own)

**Comment by gnidan** [comment]
Thanks!

**Pull request merged**